.coveragerc

```diff
@@ -1,3 +1,13 @@
 [run]
 source = openedx_webhooks
 branch = true
+omit =
+    */conftest.py
+    */test_*.py
+    openedx_webhooks/github/dispatcher/rules/github_activity.py
+    openedx_webhooks/github/models.py
+    openedx_webhooks/jira/tasks.py
+    openedx_webhooks/lib/edx_repo_tools_data/utils.py
+    openedx_webhooks/lib/jira/client.py
+    openedx_webhooks/webhook_confs.py
+    openedx_webhooks/worker.py
```

Comment:
None of these modules are very interesting, they fall into one of two categories:

* They are façades to third party API clients
* They call on other tested modules

I think at the time that they do more heavy lifting, we can remove the exceptions so they don't report false positives in terms of needing test coverage.